app/views/collections/form.html.erb

```diff
@@ -30,69 +30,53 @@
     <%# The panes below need a form in order to render their contents. This provides a "fake" form to use for that purpose. %>
     <%# TabListComponent will render the actual form. %>
     <% form_with(model: @collection_form) do |form| %>
-      <% component.with_pane do %>
-        <%= render Collections::Edit::PaneComponent.new(tab_name: :details, label: t('collections.edit.panes.details.label'), active_tab_name:) do %>
-          <%= render Elements::Forms::TextFieldComponent.new(form:, field_name: :title, label: t('collections.edit.fields.title.label'), tooltip: t('collections.edit.fields.title.tooltip_html'), container_classes: 'pane-section') %>
-          <%= render Elements::Forms::TextareaFieldComponent.new(form:, field_name: :description, label: t('collections.edit.fields.description.label'), tooltip: t('collections.edit.fields.description.tooltip_html'), container_classes: 'pane-section') %>
-          <%= render NestedComponentPresenter.for(form:, field_name: :contact_emails, model_class: ContactEmailForm, form_component: Collections::Edit::ContactEmailsComponent, hidden_label: true, bordered: false, single_field: true, fieldset_classes: 'pane-section') %>
-        <% end %>
+      <% component.with_collection_pane(tab_name: :details, label: t('collections.edit.panes.details.label'), active_tab_name:) do %>
```

Comment:
Now, instead of the pattern being:

```ruby
component.with_pane do # no context at all here
  render Collections::Edit::PaneComponent.new(lots:, of:, args:, here) # This is always the same pane component in the work context and always the same in the collection context, which suggests we have distinct pane component types already, just undeclared ones.
  # pane content here
end
```

It's now this:

```ruby
component.with_pane(lots:, of:, args:, here) do # no context at all here
  # pane content here
end
```

Reply:
An alternative we could also consider is instead of typed slot components, a lambda slot component could sniff the `model` in `TabListComponent` and render the appropriate component if we're worried about coupling.